src/components/graph/menus/create-node-menu.js

```diff
@@ -203,6 +204,7 @@ const CreateNodeMenu = ({
         CREATE_MODIFICATION_NODE: {
             onRoot: true,
             id: 'createNetworkModificationNode',
+            disabled: mapDataLoading,
```

Comment:
Why do we need to do this check ? Actualy it create an empty not built node somewhere, when does this create a mapDataLoading problem ?

Reply:
Removed